packages/x-date-pickers/src/PickersLayout/usePickerLayout.tsx

```diff
@@ -55,7 +55,11 @@ const usePickerLayout = <TValue extends PickerValidValue>(
 
   // Action bar
   const ActionBar = slots?.actionBar ?? PickersActionBar;
-  const actionBarProps = useSlotProps({
+  const {
+    // PickersActionBar does not use it and providing it breaks memoization
```

Comment:
great find! 👍🏼

Reply:
Strictly speaking it was forwarded to `PickersActionBarRoot` and so accessible through style overrides
But not documented so I think we are good.

Reply:
Great clarification. 👍 
True, but exactly, it was not even typed like it was being passed. 🙈